ts/src/base/Exchange.ts

```diff
@@ -2722,6 +2722,12 @@ export default class Exchange {
         return res === 0;
     }
 
+    safeNumberOmitZero (obj: object, key: IndexType, defaultValue: Num = undefined): Num {
+        const value = this.safeString (obj, key);
+        const final = this.parseNumber (this.omitZero (value));
+        return (final === undefined) ? defaultValue : final;
```

Comment:
why we need this ternary?

Reply:
how do you suggest otherwise, because `defaultValue` can be either numeric or string, so we can't 'safely' do `.safeString (obj, key, defaultValue)` on first line

Reply:
ok ok you're right